.github/workflows/CondaLock.yml

```
@@ -37,39 +37,37 @@ jobs:
       # Could run as single step in parallel, but would complicate logs...
       - name: Run conda-lock ${{ matrix.IMAGE }}
         run: |
-          cd ${{ matrix.IMAGE }}
-          if [ ${{ matrix.IMAGE }} = "base-notebook" ]; then
-            conda-lock lock -f environment.yml -p linux-64
-          elif [ ${{ matrix.IMAGE }} = "pangeo-notebook" ]; then
-            conda-lock lock -f environment.yml -f ../base-notebook/environment.yml -p linux-64
-          else
-            conda-lock lock -f environment.yml -f ../pangeo-notebook/environment.yml -f ../base-notebook/environment.yml -p linux-64
-          fi
-          conda-lock render -k explicit
-          ../generate-packages-list.py conda-linux-64.lock > packages.txt
-
-      - name: Upload lockfiles
-        uses: actions/upload-artifact@v4
-        with:
-          name: ${{ matrix.image }}
-          path: ${{ matrix.image }}
+          cd base-notebook
+          conda-lock lock -p linux-64 -p linux-aarch64 -p osx-64 -p osx-arm64
+	        conda-lock render -k explicit -p linux-64
+	        ../generate-packages-list.py conda-linux-64.lock > packages.txt
```

Remove tabs to fix leading indentation
```suggestion
          conda-lock lock -p linux-64 -p linux-aarch64 -p osx-64 -p osx-arm64
          conda-lock render -k explicit -p linux-64
          ../generate-packages-list.py conda-linux-64.lock > packages.txt
```